daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -331,6 +334,12 @@ commandParser = subparser $ fold
         <$> ledgerFlags (ShowJsonApi False)
         <*> many (argument str (metavar "ARG" <> help "Extra arguments to navigator."))
 
+    ledgerMeteringReportCmd = LedgerMeteringReport
```

Comment:
Other flags default to a human readable output and then have a `--json` flag. i think it would be nice to keep that here.

Reply:
How about pretty print by default and add `--compact-output` (matching `jq`’s flag name) to not pretty print?

Reply:
Done:

```
DAML> daml-helper ledger metering-report --host localhost --port 6865 --from "2022-01-22T00:00:00Z" --compact-output
{"from":"2022-01-22T00:00:00Z","participant":"participant1","toActual":"2022-01-24T11:27:01.308061Z","applications":[{"application":"app1","events":100},{"application":"app2","events":200}]}

DAML> daml-helper ledger metering-report --host localhost --port 6865 --from "2022-01-22T00:00:00Z"
{
    "from": "2022-01-22T00:00:00Z",
    "participant": "participant1",
    "toActual": "2022-01-24T11:27:05.718135Z",
    "applications": [
        {
            "application": "app1",
            "events": 100
        },
        {
            "application": "app2",
            "events": 200
        }
    ]
}
```